cf/commands/application/start_test.go

```diff
@@ -152,8 +152,8 @@ var _ = Describe("start command", func() {
 		logRepo = new(logsfakes.FakeRepository)
 		logMessages.Store([]logs.Loggable{})
 
-		closeWait := sync.WaitGroup{}
```

Comment:
**question/non-blocking** Any theory as to why the waitgroup doesn't work here and the mutex does 🤔 ? It looks fine.

Reply:
I do not see any particular reason why WaitGroup would not work here. There might be some sort of race condition or another strange thing.

Reply:
The `cf` folder is extremely old (last piece of CAPI v2 in this repo) and should be removed as soon as we rework the plugin interface. We've avoided making any changes in this part of the codebase for the last five years.

Reply:
Going to resolve this since there as been no movement in the last week